### PR TITLE
Secured admin access for blog editing, fixed add blog button, so now it is clickable on the whole width

### PR DESCRIPTION
**Pull request by Sanyi0001**

### CODE REVIEW

**Review by awqfrxx — APPROVED** [review]
works

**Review by AlexGit123 — APPROVED** [review]
Looks good

File: app/Http/Controllers/BlogController.php [hunk 1]
```diff
@@ -71,7 +71,7 @@ public function store(Blog $blog, Request $request)
      */
     public function show(Blog $blog)
     {
-        //
+        return redirect(route('blogs.index'));
```
Comment (Koreyoshi159) [inline app/Http/Controllers/BlogController.php:74]: what is this ????

File: resources/views/blogs/index.blade.php [hunk 2]
```diff
@@ -14,7 +14,7 @@
                 Use the filters to search for what you want
             </p>
         </div>
-        <button class="button is-light is-link">
+        <button class="button is-light is-link" onclick="window.location.href='/blogs/create'">
```
Comment (Koreyoshi159) [inline resources/views/blogs/index.blade.php]: This is not how it should work, then just remove the button and make the "<a>" a button using the class

**Review by Koreyoshi159 — CHANGES_REQUESTED** [review]
almost

**Pull request merged**